docs/user-guides/cdc-ingestion.md

```diff
@@ -8,4 +8,435 @@ menu:
         parent: User Guides
         weight: 400
 ---
-# CDC Ingestion
\ No newline at end of file
+# CDC Ingestion
+CDC stands for Change Data Capture, which is a broad concept, as long as it can capture the change data, it can be called CDC. [Flink CDC](https://github.com/ververica/flink-cdc-connectors) is a Log message-based data capture tool, all the inventory and incremental data can be captured. Taking MySQL as an example, it can easily capture Binlog data through Debezium and process the calculations in real time to send them to the Arctic data lake. The Arctic data lake can then be queried by other engines.
```

Comment:
Arctic's positioning is Lakehouse management service, `Arctic data lake` is an inaccurate statement.I suggest changing it directly to "data lake".